packages/dds/tree/src/feature-libraries/editable-tree/editableTreeContext.ts

```diff
@@ -130,14 +126,14 @@ export class ProxyContext implements EditableTreeContext {
 
 	/**
 	 * @param forest - the Forest
-	 * @param transactionCheckout - the Checkout applied to a transaction, not required in read-only usecases.
+	 * @param runTransaction - a function which executes a given transaction, returning true if it was successful or false if aborted. Not required in read-only use-cases.
 	 */
 	constructor(
 		public readonly forest: IEditableForest,
-		private readonly transactionCheckout?: TransactionCheckout<
-			DefaultEditBuilder,
-			DefaultChangeset
-		>,
+		private readonly runTransaction?: (
```

Comment:
I don't think this code needs a way to roll back, or do grouping. I think just passing in the `editor` would be more inline with what this code actually wants to do. Its use of transactions was just an artifact of how the olds API forced it to work

Reply:
Great, that will be much cleaner.